bindings/python/pyproject.toml

```diff
@@ -70,8 +70,22 @@ cache-keys = [
   { file = "../../core/**/*.rs" },
 ]
 
+[tool.ruff]
+line-length = 88
```

Comment:
I would vote for 120 because our screen can view these many characters with ease.

Reply:
[PEP-8](https://peps.python.org/pep-0008/#maximum-line-length) recommends 80 chars and black uses 88 by default, so I chose 88